tox.ini

```
@@ -1,8 +1,7 @@
 [tox]
 skipsdist=True
 usedevelop=True
-
-envlist=py{36}-dj{111}-{sqlite,postgres}
+envlist=lint,py{36}-dj{111,22}-{sqlite,postgres}
```

```suggestion
envlist=lint,py{36}-dj{111}-{sqlite,postgres}
```

For now let's remove 2.2 from the default set of `tox` envs until we make the tests pass.

---

Made changes such that tests pass